R/scale-colour.r

```diff
@@ -3,7 +3,12 @@
 #' Colour scales for continuous data default to the values of the
 #' `ggplot2.continuous.colour` and `ggplot2.continuous.fill` options. These
 #' [options()] default to `"gradient"` (i.e., [scale_colour_gradient()] and
-#' [scale_fill_gradient()])
+#' [scale_fill_gradient()]). Note that both continuous and binned scales
+#' are configured with the same options values. In other words,
+#' the option `ggplot2.continuous.colour` controls the behavior of both
+#' `scale_colour_continuous()` and `scale_colour_binned()`. To manually set
```

Comment:
Wouldn't it be more correct to say that the binned scale falls back to `ggplot2.continuous.colour` if `ggplot2.binned.colour` is not set?